en/getting_started/px4_basic_concepts.md

```diff
@@ -28,23 +28,70 @@ You can read in depth about heading in [Flight Controller Orientation](../config
 
```

Comment:
FYI, the PX4 concepts is a bit of a mishmash of general concepts and overview/features list. This updates it more towards being an overview doc that references other concepts docs. As such it includes new section on Flight Modes and this update for Failsafes. I could have added a separate doc for failsafe at this point if you thought worth highlighting as a standalone concept (I thought overkill) ?

Reply:
I like this update very much. I think flight modes and failsafes are both important basic concepts, every new user would be interested in.